subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/ArgsProvider.kt

```diff
@@ -49,4 +51,14 @@ class BundleArgsProvider(
             return result
         }
     }
+
+    override fun getOptionalIntArgument(name: String): Int? {
+        val defaultValue = Int.MIN_VALUE
+        val result = bundle.getInt(name, defaultValue)
+        return if (result == defaultValue) {
```

Comment:
Why don't we check it by `containsKey`?

Reply:
i think it is a bit more safe that way

from bundle.getInt
```
 catch (ClassCastException e) {
            typeWarning(key, o, "Integer", defaultValue, e);
            return defaultValue;
        }
```

and no default value means default is 0, so we will get 0 result if pass something not convertable to int